Tinylog/Application/AppDelegate.swift

```diff
@@ -19,16 +19,17 @@
     var window: UIWindow?
 
     /// Access core data managed object context.
-    let coreDataManager = CoreDataManager(model: "Tinylog")
-
-    var applicationCoordinator: ApplicationCoordinator!
+    private let coreDataManager = CoreDataManager(model: "Tinylog")
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sger/Tinylog/blob/fcd3bd9795ffab4cd3746fabd6ba114cd64daeee/Tinylog/Application/AppDelegate.swift#L23"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AppDelegate.swift:23</code></td>
    </tr>
  </tbody>
</table>